cloud_AWS/terraform/module/examples/multiple-vpc-regions/multi_region_data.ini

```diff
@@ -0,0 +1,11 @@
+[first-region]
+region =
+external_id =
```

Comment:
`external_id` and `plan_id` are always the same for all regions. They should be in some `global` (or `defaults` section.

Reply:
Moved to [DEFAULT] section of INI file